src/api/video.ts

```diff
@@ -4,22 +4,25 @@ import firebase from "firebase/app";
 export interface GetVideoTokenProps {
   userId: string;
   roomName: string;
+  onError?: (msg: string) => void;
+  onFinish?: () => void;
```

Comment:
As above RE: not creating legacy callback APIs when we already use `Promise`s + re-throwing the error in the `.catch` after the `Bugsnag.notify`

Reply:
https://github.com/sparkletown/sparkle/pull/1299#discussion_r626555245